src/windowmanagement/HelpMenu.tsx

```diff
@@ -1,26 +1,144 @@
-import { Button, Menu } from '@mantine/core';
+import { Anchor, Button, Grid, Image, Menu, Modal, Text, Title } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import {
+  BookIcon,
+  ConsoleIcon,
+  ExitAppIcon,
+  FeedbackIcon,
+  InformationCircleOutlineIcon,
+  OpenInBrowserIcon,
+  RouteIcon
+} from '@/icons/icons';
+import { useAppSelector } from '@/redux/hooks';
 
 export function HelpMenu() {
+  const [opened, { open, close }] = useDisclosure(false);
+  const luaApi = useOpenSpaceApi();
+  const openSpaceVersion = useAppSelector((state) => state.version.openSpaceVersion);
+
+  async function toggleLuaConsole() {
+    if (!luaApi) {
+      return;
+    }
+    const visible = (await luaApi.propertyValue('LuaConsole.IsVisible')) as
+      | boolean
+      | undefined;
```

Comment:
use `useGetBoolPropertyValue` instead (above the function) ? Why get it from the engine when we already have it in redux?